content/news/mar2020-openff-nih-funding

```diff
@@ -11,9 +11,9 @@ author: "Michael Shirts"
 
 <br>
 
-We are excited to announce that the NIH Institute of General Medical Sciences has funded our Focused Technology Research & Development R01 proposal, enabling the Open Force Field Initiative to extend its work for small molecule force fields to also include biopolymers. 
+We are excited to announce that the NIH Institute of General Medical Sciences has funded our Focused Technology Research & Development R01 proposal, enabling the Open Force Field Initiative to partner with the biopolymer force field community and extend our infrastructure and force fields to also cover biopolymers.
 This is a collaborative NIH grant, with $2.5M over four years shared over the five main academic sites (University of Colorado Boulder, Memorial Sloan Kettering Cancer Center, University of California, Irvine, University of California, San Diego, and University of California, Davis), and includes significant funded collaborations with the MolSSI QCArchive and the AMBER community (via David Cerutti at Rutgers). The submitted proposal text can be found [here](https://openforcefield.org/science/downloads/proposals/2019-03-05%20Open%20Force%20Field%20Initiative%20R01%20science.pdf).
```

Comment:
Can we instead emphasize the direct cost $/year? We don't want to sound _rich_---we want to make it clear that this is only a few person-slots.